Parsers/Cloudflare/Cloudflare

```diff
@@ -0,0 +1,216 @@
+// Usage Instruction :
```

Comment:
Can you please update filename to have a .txt extension.

Reply:
updated